manifests/params.pp

```diff
@@ -6,7 +6,13 @@
   $repo_name     = 'google-chrome'
 
   case $::osfamily {
-    'RedHat', 'Suse': {
+    'RedHat' : {
+      if $::os[release][major] == '6' {
```

Comment:
It's been a while since I did any Puppet / Ruby. Don't the hash keys need to be quoted?

CI Build failed with:

`Error: ::os is not a hash or array when accessing it with release at /etc/puppet/modules/google_chrome/manifests/params.pp:10`